VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -88,6 +88,7 @@ export abstract class WaitingRoomBaseDirective {
     displayStartPrivateConsultationModal: boolean;
     displayJoinPrivateConsultationModal: boolean;
     conferenceStartedBy: string;
+    shouldUpdateHostShowVideo = false;
```

Comment:
I find the name a bit confusing. Didn't we have one previously that behaved in a similar way?

Reply:
> I find the name a bit confusing. Didn't we have one previously that behaved in a similar way?

We had one called dualHostHasSignalledToJoinHearing